### PR TITLE
qownnotes: Add version 21.6.1

### PR DESCRIPTION
**Pull request by rdmcgregor**

### CODE REVIEW

**Comment by rdmcgregor** [comment]
What needs to be fixed?  Is there something I have missed?

File: bucket/qownnotes.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "21.6.0",
+    "description": "",
```
Comment (Ash258) [inline bucket/qownnotes.json:3]: ???

File: bucket/qownnotes.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://www.qownnotes.org/",
+    "license": "GPL-2.0",
```
Comment (Ash258) [inline bucket/qownnotes.json:5]: Invalid SPDX identifier

File: bucket/qownnotes.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/pbek/QOwnNotes/releases/download/v21.6.0/QOwnNotes.zip",
+    "hash": "8BCF6460A1A5DA4F483E8B883D6CC86B82C5C60D568064DC603A3DC5A1E17FD7",
```
Comment (Ash258) [inline bucket/qownnotes.json:7]: Checkver was not used

**Review by Ash258 — CHANGES_REQUESTED** [review]
image

File: bucket/qownnotes.json (hunk 1 above)
Comment (rdmcgregor) [reply bucket/qownnotes.json]: fixed missing description

File: bucket/qownnotes.json (hunk 2 above)
Comment (rdmcgregor) [reply bucket/qownnotes.json]: fixed license id

File: bucket/qownnotes.json (hunk 3 above)
Comment (rdmcgregor) [reply bucket/qownnotes.json]: fixed

**Comment by rdmcgregor** [comment]
Updated, didn't use portable option for execution because that doesn't seem like the best choice for executing this app after installing with scoop.

**Review by Ash258 — CHANGES_REQUESTED** [review]
Use portable mode.

**Comment by rdmcgregor** [comment]
Migrated to portable mode and ran url and version checks

File: bucket/qownnotes.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/pbek/QOwnNotes/releases/download/v21.6.1/QOwnNotes.zip",
+    "hash": "d79468f688e07c91d0657a4c74400a49b979de1eab38a92f6ec8eb025604e908",
+    "bin": "QOwnNotes.exe",
+    "shortcuts": [
+        [
+            "QOwnNotesPortable.bat",
+            "QOwnNotes"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/qownnotes.json:14]: Where is persist??

**Review by Ash258 — DISMISSED** [review]

**Comment by issaclin32** [comment]
I updated the manifest by myself because this PR hasn't been updated for a while.

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**